torch/_inductor/kernel/mm.py

```diff
@@ -199,3 +201,18 @@ def tuned_addmm(inp, mat1, mat2, *, alpha=1, beta=1, layout=None):
     return autotune_select_algorithm(
         "addmm", choices, [inp_expanded, mat1, mat2], layout
     )
+
+
+def tuned_mixed_mm(mat1, mat2, mat2_dtype):
+    m, n, k, layout, mat1, mat2 = mm_args(mat1, mat2, layout=None)
+    # options to tune from
+    choices = []
+    b_prologue_cast_type = f"tl.{mat2_dtype}".replace("torch.", "")
+    for config in mm_configs(m, n, k):
+        mm_template.maybe_append_choice(
```

Comment:
Will this work on CPU?

Reply:
no, its similar to tuned_int_mm

Reply:
do you think it should be gated so it doesn't match on CPU?

Reply:
I think it would be fine to just fall back to the eager ops on CPU, but it should at least run without error and be tested.

Reply:
ok done